perfkitbenchmarker/linux_benchmarks/netperf_pps_benchmark.py

```diff
@@ -147,43 +147,38 @@ def RunNetperfAggregate(vm, server_ips):
   """
 
   # setup remote hosts file
-  vm.RemoteCommand("cd %s && rm remote_hosts"
-                   % (netperf.NETPERF_EXAMPLE_DIR))
+  vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && rm remote_hosts")
   ip_num = 0
   for ip in server_ips:
-    vm.RemoteCommand("cd %s && echo 'REMOTE_HOSTS[%d]=%s' >> remote_hosts"
-                     % (netperf.NETPERF_EXAMPLE_DIR, ip_num, ip))
+    vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && "
+                     f"echo 'REMOTE_HOSTS[{ip_num}]={ip}' >> remote_hosts")
     ip_num += 1
 
-  vm.RemoteCommand("cd %s && echo 'NUM_REMOTE_HOSTS=%d' >> remote_hosts"
-                   % (netperf.NETPERF_EXAMPLE_DIR, len(server_ips)))
-  vm.RemoteCommand('cd %s && export PATH=$PATH:.'
-                   % (netperf.NETPERF_EXAMPLE_DIR))
+  vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && "
+                   f"echo 'NUM_REMOTE_HOSTS={len(server_ips)}' >> remote_hosts")
+  vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && export PATH=$PATH:.")
 
   # allow script to be executed and run script
-  stdout, stderr = vm.RemoteCommand("cd %s && export PATH=$PATH:. && chmod "
+  stdout, stderr = vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && "
+                                    "export PATH=$PATH:. && chmod "
                                     "+x runemomniaggdemo.sh && "
-                                    "./runemomniaggdemo.sh"
-                                    % (netperf.NETPERF_EXAMPLE_DIR),
+                                    "./runemomniaggdemo.sh",
                                     ignore_failure=True, should_log=True,
                                     login_shell=False, timeout=1200)
 
   # print out netperf_tps.log to log
-  stdout_1, stderr_1 = vm.RemoteCommand("cd %s && cat netperf_tps.log" %
-                                        (netperf.NETPERF_EXAMPLE_DIR),
+  stdout_1, stderr_1 = vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && cat netperf_tps.log",
```

Comment:
use single quotes.  I would say so above but that introduces some code churn

Reply:
Changed to single quotes